### PR TITLE
ZBUG-2863: Problems when viewing admin console at higher resolutions

### PR DESCRIPTION
**Pull request by lavegupta**

### CODE REVIEW

File: WebRoot/js/zimbraAdmin/search/model/ZaSearch.js [hunk 1]
```diff
@@ -146,8 +146,14 @@ ZaSearch.searchDirectory = function(params) {
         ZaSearch._currentQuery = "";
     }
 
+    var EXTENDEDRESULTSPERPAGE;
+    if(params.offset == 0) {
+        var tempM = 1 + parseInt((document.documentElement.clientHeight - 100)/1000);
```
Comment (silentsakky) [inline WebRoot/js/zimbraAdmin/search/model/ZaSearch.js:151]: use parseInt with base parameter

File: WebRoot/js/zimbraAdmin/search/controller/ZaSearchListController.js [hunk 2]
```diff
@@ -366,35 +366,36 @@ function(params) {
...
+            showBusy:true,
+            busyId:busyId,
+            busyMsg:ZaMsg.BUSY_SEARCHING,
+            skipCallbackIfCancelled:false,
+            sortBy:params.sortBy,
+            sortAscending:this._currentSortOrder,
+            attrs:ZaSearch.standardAttributes,
+            controller:this,
+            scrollType:"isAliasSearch",
+            postCallback: postCallback
```
Comment (jiteshsojitra) [inline WebRoot/js/zimbraAdmin/search/controller/ZaSearchListController.js:380]: indentation to use consistent spaces or tab key.

File: WebRoot/js/zimbraAdmin/search/model/ZaSearch.js [hunk 3]
```diff
@@ -146,8 +146,14 @@ ZaSearch.searchDirectory = function(params) {
         ZaSearch._currentQuery = "";
     }
 
+    var customResultValue;
+    if(params.isInitialRequest) {
+        var multiplier = 1 + parseInt((document.documentElement.clientHeight - 100)/1000, 0);
```
Comment (k-kato) [inline WebRoot/js/zimbraAdmin/search/model/ZaSearch.js:151]: * Is `document.documentElement.clientHeight` a string? If it is a number, `parseInt` is unnecessary.
* If parseInt is necessary, why is radix (second param) `0`, not `10` ?

Comment (k-kato) [reply WebRoot/js/zimbraAdmin/search/model/ZaSearch.js:151]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt
```
Syntax
parseInt(string)
parseInt(string, radix)
```

Comment (silentsakky) [reply WebRoot/js/zimbraAdmin/search/model/ZaSearch.js:151]: good catch it should be 10

File: WebRoot/js/zimbraAdmin/search/controller/ZaSearchListController.js (hunk 2 above)
Comment (lavegupta) [reply WebRoot/js/zimbraAdmin/search/controller/ZaSearchListController.js]: Fixed

**Review by k-kato — APPROVED** [review]

**Review by damini-vashishtha — APPROVED** [review]

**Review by silentsakky — APPROVED** [review]

**Pull request merged**